### PR TITLE
Add runtime configurable correlation log support to MI

### PR DESCRIPTION
**Pull request by pasindu-sandima**
support enable/disable correlation logs without a server restart.

### CODE REVIEW

File: components/org.wso2.micro.integrator.extensions/org.wso2.micro.integrator.management.apis/src/main/java/org/wso2/micro/integrator/management/apis/ConfigsResource.java [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+            case Constants.HTTP_PUT: {
+                response = handlePut(axis2MessageContext);
+                break;
+            }
+            default: {
+                response = Utils.createJsonError(
+                        "Unsupported HTTP method, " + httpMethod + ". Only GET and " + "PUT methods are supported",
+                        axis2MessageContext, Constants.BAD_REQUEST);
+                break;
+            }
```
Comment (sachithKay) [inline components/org.wso2.micro.integrator.extensions/org.wso2.micro.integrator.management.apis/src/main/java/org/wso2/micro/integrator/management/apis/ConfigsResource.java:84]: formatting

**Comment by sachithKay** [comment]
there are formatting issues. Please recheck formatting

File: pom.xml [hunk 2]
```diff
@@ -1449,7 +1449,7 @@
         <carbon.kernel.imp.pkg.version>[4.5.0,5.0.0)</carbon.kernel.imp.pkg.version>
         <version.jaxb.api>2.4.0-b180830.0359</version.jaxb.api>
 
-        <synapse.version>2.1.7-wso2v282</synapse.version>
+        <synapse.version>2.1.7-wso2v288-SNAPSHOT</synapse.version>
```
[the remark this answers is not in the document]
Comment (pasindu-sandima) [reply pom.xml:1452]: Okay

**Comment by pasindu-sandima** [comment]
Tracked : https://github.com/wso2/micro-integrator/pull/2732

**Pull request closed without merging**